.github/workflows/release.yaml

```diff
@@ -1,16 +1,16 @@
-name: wheels
+name: release
 
 on:
   push:
     tags:
-      # 'r[0-9]+.[0-9]+.[0-9]+' to match semantic version tag, e.g. r2.0.8
-      - "r[0-9]+.[0-9]+.[0-9]+"
+      # 'v[0-9]+.[0-9]+.[0-9]+' to match semantic version tag, e.g. v2.0.8
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
@kylecarow , I'm thinking I like the solution Nick came up with in ALTRIOS:
https://github.com/NREL/altrios-private/blob/cf47f12f24589ad01a3f71dba4c0af272be6c12e/.github/workflows/wheels.yaml#L5